changelog/fix-8059-qit-security-test-failed-in-develop-branch

```diff
@@ -0,0 +1,4 @@
+Significance: patch
+Type: fix
+
+Fix QIT security tests errors.
```

Comment:
Can prefix this with a `Comment:` to hide it from the changelog as it is not relevant to external folks.

Reply:
Done in https://github.com/Automattic/woocommerce-payments/pull/8060/commits/0b88bef51d70337c267f1519a81710d55a0e833b thanks!